website/docs/sdk-reference/react.mdx

```diff
@@ -977,6 +987,48 @@ If you do not want to expose the SDK key or the content of the config JSON file,
 
 Also, we recommend using [confidential targeting comparators](../targeting/targeting-rule/user-condition.mdx#confidential-text-comparators) in the Targeting Rules of those feature flags that are used in the frontend/mobile SDKs.
 
+## Using multiple providers
+
+To access a feature flag's value, your code needs to refer to that Config where that particular feature flag is located. This is what a ConfigCatProvider does for you; it refers to the Config by specifying the Config's sdkKey.
+
+If your code needs access to feature flags from multiple Configs, you will need multiple ConfigCatProviders - each pointing to a Config by specifying that particular Config's sdkKey.
+
+And when your code wants to access a particular feature flag, it should refer to the specific ConfigCatProvider that is associated with the Config where that particular feature flag is located. This is exactly what the providerId allows your code to do: to refer to the specific ConfigCatProvider that is associated with the feature flag it wants to access.
+
+```tsx
+const CC_PROVIDERID = {
+  COMMON: "COMMON",
+  FRONTEND: "FRONTEND"};
+
+<ConfigCatProvider sdkKey={"sdkKey-01"} id={CC_PROVIDERID.COMMON}> 
+  <ConfigCatProvider sdkKey={"sdkKey-02"} id={CC_PROVIDERID.FRONTEND}>
+      ...
+      const myFeatureInCommon = useFeatureFlag("myFeatureInCommon", false, void 0, CC_PROVIDERID.COMMON)
+
+      const myFeatureInFrontEnd = useFeatureFlag("myFeatureInFrontEnd", false, void 0, CC_PROVIDERID.FRONTEND)
+      ...
+  </ConfigCatProvider>
+</ConfigCatProvider>
+```
+
+When you are using multiple providers and they are not embed in each other it is not necessary to specify the `id` attribute:
+
+```tsx
+<ConfigCatProvider sdkKey={"sdkKey-01"}>
+  ...
+  const myFeatureInCommon = useFeatureFlag("myFeatureInCommon", false);
+  ...
+</ConfigCatProvider>
+
+<ConfigCatProvider sdkKey={"sdkKey-02"}>
+  ...
+  const myFeatureInFrontEnd = useFeatureFlag("myFeatureInFrontEnd", false);
+  ...
+</ConfigCatProvider>
+```
+
+If you are using higher order components you can set `providerId` parameter in `withConfigCatClient` function.
```

Comment:
```suggestion
If you are using higher order components, you can specify the `providerId` parameter in the `withConfigCatClient` function.
```

Reply:
The missing comma after the if clause is a grammatical error. See https://www.ego4u.com/en/cram-up/writing/comma?10

Also, please avoid referencing to identifiers like "set `providerId` parameter". It should be either "set the `providerId` parameter" or simply "set `providerId`".

I'll update my suggestion accordingly.